frontend/app/templates/components/home-menu.hbs

```diff
@@ -0,0 +1,22 @@
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed"
+      data-toggle="collapse" data-target="#menu-collapse">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      {{ link-to 'Home' 'index' class="navbar-brand" }}
```

Comment:
It's there in the latest guide as an acceptable format for inline links. Doesn't say it will be deprecated. https://guides.emberjs.com/v2.5.0/templates/links/
I've fixed everything else, I think.